index.js

```diff
@@ -1025,13 +890,6 @@ Client.prototype._currentReconcile = function (callback) {
 
         self.state.reconcileStamp = underscore.now() + (self.state.properties.days * msecs.day)
         if (self.options.verboseP) self.state.reconcileDate = new Date(self.state.reconcileStamp)
-
-        self._updateRules(function (err) {
-          if (err) self._log('_updateRules', { message: err.toString() })
-
-          self._log('_currentReconcile', { delayTime: msecs.minute })
-          callback(null, self.state, msecs.minute)
```

Comment:
We still need this callback to keep contribution going.

Reply:
very good catch..

Reply:
re-added